app/routes/authenticated.js

```diff
@@ -42,7 +42,10 @@ const AuthenticatedRoute = Route.extend(AuthenticatedRouteMixin, {
       accountId: this.get("org.selected.id")
     };
     triggerAnalytics('login', data);
-    chat.setUser(user.get("email"), user.get("crispHash"))
+    chat.setUserEmail(user.get("email"), user.get("crispHash"))
+    const org = this.get("org.selected.data.name")
+    const company = org || user.get("email").split("@").pop().trim()
```

Comment:
`user` also has a company field, check that first.

Reply:
`user.company`

Reply:
oh, that is not beings sent in the API response, will add that and use it